### PR TITLE
truyenqqhot.com

### PR DESCRIPTION
**Pull request by tachibana-shin**
# Creating the pull request

> Please include a summary of the change and which issue is fixed\
> If the related issue does not exist in our repository, please create it before making pull request\
> It is highly recommended to use our [Web Reporting Tool](https://kb.adguard.com/en/technical-support/reporting-tool) instead of creating an issue on GitHub directly\
> Please note, that we verify every pull request manually, so it may take time to apply it

### To avoid invalid pull requests, please check and confirm following terms

- [x] This is not an ad/bug report;
- [x] My code follows the [guidelines](https://github.com/AdguardTeam/AdguardFilters/blob/master/CONTRIBUTING.md) and [syntax](https://kb.adguard.com/general/how-to-create-your-own-ad-filters) of this project;
- [x] I have performed a self-review of my own changes;
- [x] My changes do not break web sites, apps and files structure.

### If the problem does not fall under any category that is listed here, please write a comment below in corresponding section

- [x] Missed ads or ad leftovers;
- [ ] Website or app doesn't work properly;
- [ ] AdGuard gets detected on a website;
- [ ] Missed analytics or tracker;
- [ ] Social media buttons — share, like, tweet, etc;
- [ ] Annoyances — pop-ups, cookie warnings, etc;
- [ ] Filters maintenance.

### Enter the issue address

https://github.com/AdguardTeam/AdguardFilters/issues/139741

#### If possible, a screenshot of a page or application should not be cropped too much. Otherwise, it is not always clear where the element is located

0. **DO NOT** upload screenshots with sexually explicit material on GitHub directly.\
 Instead, upload it to a third-party image hosting and post the link to it without preview (!) here.\
 Also, mention if the link leads to NSFW content;

1. Add screenshots of the problem. You can drag and drop images or paste them from clipboard

    Use `<details> </details>` tag to hide screenshots under the spoiler;

2. Describe the issue in detail until it is absolutely clear from the screenshot what the problem is

    You can also indicate any other information that you think the developers should know.

**Warning:** Please remove personal information before uploading screenshots!

1. Your comment

this site often randomly changes the class and id of their ad elements so I'm going to make a pull request that includes the ad filters that are already appearing

2. Screenshots

<details>

<summary>Screenshot 1:</summary>

Ảnh chụp màn hình 2023-01-10 211434

</details>

<details>

<summary>Screenshot N:</summary>

</details>

### Terms

- [x] By submitting this issue, I agree that pull request does not contain private info and all conditions are met

### CODE REVIEW

File: BaseFilter/sections/foreign.txt [hunk 1]
```diff
@@ -10650,6 +10650,11 @@ phimsexmoi.biz###text-2[style*="left: 0px; top: 0px;"]
 nhaccuatui.com###eventBannerLeft
 nhaccuatui.com###eventBannerTop
 nhaccuatui.com##div[class^="adv_"]
+truyenqqhot.com###ad_info_top
+truyenqqhot.com###ad_info
+truyenqqhot.com###ads_info
+truyenqqhot.com###ad_infos
+truyenqqhot.com##.text_ads
```
Comment (Alex-302) [inline BaseFilter/sections/foreign.txt:10657]: It seems `truyenqqhot.com##div[id^="ad_"]` is enough.

Comment (tachibana-shin) [reply BaseFilter/sections/foreign.txt:10657]: that filter looks dangerous. Recently they have an id called `ad_poster` which contains some announcements about the change

Comment (Alex-302) [reply BaseFilter/sections/foreign.txt:10657]: What about `truyenqqhot.com##div[id^="ad_info"]`?

Comment (zloyden) [reply BaseFilter/sections/foreign.txt:10657]: Can you provide the URL?
Check this `truyenqqhot.com##div[id^="ad_"]:not([id*="_poster"])` or `truyenqqhot.com##div[id^="ad_"][id*="info"]`

Comment (tachibana-shin) [reply BaseFilter/sections/foreign.txt:10657]: > What about `truyenqqhot.com##div[id^="ad_info"]`?

they have ad ids named `ads_info`

Comment (tachibana-shin) [reply BaseFilter/sections/foreign.txt:10657]: I think I'll shorten the filter to
```
truyenqqhot.com##div[id^="ad_"]:not([id*="_poster"])
truyenqqhot.com###ads_info
truyenqqhot.com##.text_ads
```

Comment (tachibana-shin) [reply BaseFilter/sections/foreign.txt:10657]: > Can you provide the URL? Check this `truyenqqhot.com##div[id^="ad_"]:not([id*="_poster"])` or `truyenqqhot.com##div[id^="ad_"][id*="info"]`

this is url https://truyenqqhot.com/truyen-tranh/sakurai-san-wa-kizuite-hoshii-8558-chap-10.html
they usually change the ids every week or even every day

File: BaseFilter/sections/foreign.txt [hunk 2]
```diff
@@ -10650,11 +10650,8 @@ phimsexmoi.biz###text-2[style*="left: 0px; top: 0px;"]
 nhaccuatui.com###eventBannerLeft
 nhaccuatui.com###eventBannerTop
 nhaccuatui.com##div[class^="adv_"]
-truyenqqhot.com###ad_info_top
-truyenqqhot.com###ad_info
-truyenqqhot.com###ads_info
-truyenqqhot.com###ad_infos
-truyenqqhot.com##.text_ads
+truyenqqhot.com##div[id^="ad_"]:not([id*="_poster"])
+truyenqqhot.com##div[id^="ads_"]
```
Comment (zloyden) [inline BaseFilter/sections/foreign.txt:10654]: I don't see where the `truyenqqhot.com##div[id^="ads_"]` rule are needed?

Comment (tachibana-shin) [reply BaseFilter/sections/foreign.txt:10654]: > I don't see where the `truyenqqhot.com##div[id^="ads_"]` rule are needed?

they have ad ids called `ads_top` sometimes `ad_top` looks like they will randomly update these id names

Comment (tachibana-shin) [reply BaseFilter/sections/foreign.txt:10654]: > I don't see where the `truyenqqhot.com##div[id^="ads_"]` rule are needed?

anyways updating the filter when they change to `ads_top` is very time consuming so i think its better to include in this pull request

**Review by zloyden — APPROVED** [review]

**Pull request merged**